libraries/Adapters/Microsoft.Bot.Builder.Adapters.Twilio/TwilioAPI.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+
+namespace Microsoft.Bot.Builder.Adapters.Twilio
+{
+    public class TwilioAPI : ITwilioApi
```

Comment:
Could we rename this to TwilioClient and ITwilioClient? This is the name that Gabo used.